resources/explorer/resultsList.css

```diff
@@ -17,8 +17,10 @@
 }
 
 #resultslisttablecontainer {
+    height: 50vh;
```

Comment:
Sure, but I'll caveat that I'm not an experienced CSS user, so my understanding might be off. :)

I think the current setup allows the div to keep growing outside of the viewport, and `flex` doesn't seem to constrain that. I just tried making that change locally and it causes the contents to continue past the bottom of the window as it did before.

By setting a height based on the viewport, we're forcing the div to have a max size regardless of its contents, which allows us to scroll in the case of overflow.

Reply:
Thanks for that explanation, Wilson.

If your goal is to force overflow scroll, I recommend setting `overflow: hidden` on `#resultslistcontainer` instead of setting `height` on `#resultslisttablecontainer`.

I notice you have also added resizing behavior to the pane. With your changes, I noticed after resize, gaps (whitespace) may start appearing. I think correctly implementing resize requires a few more structural changes, and I would recommend it as a separate PR.

Note: I don't normally work on this project. I don't have the dev environment set up. So I was working with an approximation of the layout. So hopefully we're seeing the same thing.

Reply:
Thanks for the feedback!  I should have time to revisit this PR at the end of the week, but feel free to take over the PR if you find some time before then.

> If your goal is to force overflow scroll, I recommend setting `overflow: hidden` on `#resultslistcontainer` instead of setting height on `#resultslisttablecontainer`

Can you elaborate on why you'd hide overflow and unset height? If I'm reading the docs correctly, setting `overflow: hidden` would clip the view of `#resultlistcontainer`'s inner contents, but it would also hide any scrollbars / prevent scrolling, which is the misbehavior we're trying to fix. According to Mozilla's docs on `overflow`, we need to set `height` or `max-height` on the block-level container to make things work. Without it, I think the div would just keep expanding and never actually trigger an overflow situation.

> I notice you have also added resizing behavior to the pane. With your changes, I noticed after resize, gaps (whitespace) may start appearing. I think correctly implementing resize requires a few more structural changes, and I would recommend it as a separate PR.

That makes sense. I'll remove the resize rule in my next set of changes.

I'm not really sure what to do with the result detail pane under the result list, because giving it 50vh just kind of seemed like a lot. Adding resize was my way of kicking that design decision down the road. Until that's done, I might just try to bottom-align that div and let it expand upward from the bottom of the screen or something.

Reply:
Np! `#resultslistcontainer` `overflow: hidden` (somewhat counterintuitively) causes `#resultslisttablecontainer` `overflow-y: scroll` to take effect.

Regarding the details pane split design, I need to look closer to form an opinion.

Reply:
I was able to verify the described fix for scrolling. I have a moment now, so I'll take over the PR (or just start a new one). Thanks for your help Wilson!

Reply:
Thanks for carrying the fix across the finish line and for the CSS tips! I'm going to close out this PR now that the issue's been resolved.